### PR TITLE
Update SceneUtils.html

### PR DESCRIPTION
**Pull request by gero3**
Adds documentation of nessecity of correct matrixworld in attach and detach.

Update in response to #6426

### CODE REVIEW

File: docs/api/extras/SceneUtils.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		parent -- The parent to attach the object from.
 		</div>
 		<div>
-		Attaches the object to the parent without the moving the object in the worldspace.
+		Attaches the object to the parent without the moving the object in the worldspace. Beware that to do this the worldMatrix needs to be updated, this can be done by calling the updateMatrixWorld method on the parent object.
```
Comment (mrdoob) [inline docs/api/extras/SceneUtils.html]: Beware that to do this the *matrixWorld* needs...

**Comment by WestLangley** [comment]
The solution we usually use is to call `object.updateMatrixWorld()` or `object.updateMatrixWorld( true )` before accessing `object.matrixWorld` inside a method. See, for example, `Object3D.getWorldPosition()`. So we could modify `attach()` and `detach()` accordingly.

The problem is that the only _correct_ solution is to update the world matrix of _all_ the object's parents, starting from the scene. We have never implemented that approach.

**Comment by benaadams** [comment]
> The problem is that the only correct solution is to update the world matrix of all the object's parents, starting from the scene. We have never implemented that approach.

Remove ```parent``` as parameter from ```detach```, use the objects parent? Traverse till top of graph hit, call ```topParent.updateMatrixWorld()```

In ```attach```, remove ```scene``` as parameter. Traverse till top of graph hit, call  ```topParent.updateMatrixWorld()``` then remove ```object``` from ```object.parent```?

**Comment by WestLangley** [comment]
> Traverse till top of graph hit, call topParent.updateMatrixWorld()

We really only want to update the world matrix of the one branch the object is on.

**Comment by benaadams** [comment]
> We really only want to update the world matrix of the one branch the object is on.

Hmm, good point, could cause bad cascading updates otherwise

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**